contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -223,13 +223,26 @@ def process_assessment_metadata(ccnode, kolibrinode):
     assessment_items = ccnode.assessment_items.all().order_by('order')
     exercise_data = json.loads(ccnode.extra_fields) if ccnode.extra_fields else {}
 
-    mastery_model = {'type' : exercise_data.get('mastery_model') or exercises.M_OF_N}
+    mastery_model = {
```

Comment:
The `mastery_model` would then go into the assessmentmetadata as before, while the `legacy_mastery_model` object would be used in the exercise.json.